packages/taquito/test/global-constants/default-global-constants-provider.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { DefaultGlobalConstantsProvider } from '../../src/global-constants/default-global-constants-provider';
+
+describe('DefaultGlobalConstantsProvider tests', () => {
+    it('DefaultGlobalConstantsProvider is instantiable', () => {
+        expect(new DefaultGlobalConstantsProvider()).toBeInstanceOf(DefaultGlobalConstantsProvider);
+    });
+
+    it('register a global constant and retrieve the value by its hash', async (done) => {
+        const hash = 'expru5X5fvCer8tbRkSAtwyVCs9FUCq46JQG7QCAkhZSumjbZBUGzb';
+        const michelineExpr = {
```

Comment:
looks like this michelineExpression is being reused a few times in the file, maybe we could initialize it once above the describe block and reuse that in the it blocks?

Reply:
Good catch :)